projects/components/src/toggle-switch/toggle-switch.component.ts

```diff
@@ -12,7 +12,7 @@ import { ToggleSwitchSize } from './toggle-switch-size';
       <mat-slide-toggle
         color="primary"
         [checked]="this.isChecked"
-        [aria-label]="this.ariaLabel ?? this.label"
+        [aria-label]="this.ariaLabel ?? this.label || 'toggle switch'"
```

Comment:
do we need to add a role ? Probably mat slide toggle might be applying it under the hood.

Reply:
Does it this necessary? I believe that we should provide the ariaLabel in all occurrences because we are going to help users with these changes and not try to force the 100%